Khan/perseus
# [Locked Figure Labels] View locked function labels

## Summary:
- Add feature flag for locked function labels
- Add labels to LockedFunctionType
- update builder
- update stories and tests
  - add a story for all locked figure labels

Issue: https://khanacademy.atlassian.net/browse/LEMS-2351

## Test plan:
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-function-labels-flag
- Confirm that the visible label is on the locked function only
- Go to the other locked figures' stories and confirm that
  the locked function does not have a visible label (to make sure
  that the flags are separating the behaviors correctly)

<img width="492" alt="image" src="https://github.com/user-attachments/assets/4e673a9e-2ebc-4145-8f21-06364ead89ca">
